lib/msal-browser/src/broker/nativeBroker/NativeMessageHandler.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { NativeConstants, NativeExtensionMethod } from "../../utils/BrowserConstants";
+import { Logger, AuthError, AuthenticationScheme } from "@azure/msal-common";
+import { NativeExtensionRequest, NativeExtensionRequestBody } from "./NativeRequest";
+import { NativeAuthError } from "../../error/NativeAuthError";
+import { BrowserAuthError } from "../../error/BrowserAuthError";
+import { BrowserConfiguration } from "../../config/Configuration";
+
+export class NativeMessageHandler {
+    private extensionId: string | undefined;
+    private logger: Logger;
+    private handshakeTimeoutMs: number;
+    private responseId: number;
+    private timeoutId: number | undefined;
+    private resolvers: object;
```

Comment:
Can you make this a map?

Pseudo code:
```typescript
type ResponseResolvers<T> {
  resolve: Function<T>,
  reject: Function
}
```

```suggestion  
    private resolvers: Map<number, ResponseResolvers>
```

Reply:
done